data/usecases.ts

```diff
@@ -0,0 +1,48 @@
+// possible icons, so far: 'user', 'arrow', 'speech' and 'check'
+
+import React, { ReactNode } from 'react';
+
+export type UseCase = {
+    icon: string;
```

Comment:
Instead of this comment above:

```
// possible icons, so far: 'user', 'arrow', 'speech' and 'check'
```

we can use union type here

```ts
icon: 'user' | 'arrow' | 'speech' | 'check';
```